docs/zh/latest/discovery/kubernetes.md

```diff
@@ -45,7 +45,7 @@ discovery:
       port: ${KUBERNETES_SERVICE_PORT}  #default ${KUBERNETES_SERVICE_PORT}
 
     client:
-      # serviceaccount token or token_file
+      # serviceaccount token or token_file token_file中应注意不要有换行符 请使用echo -n token > token_file 来生成此文件
```

Comment:
```suggestion
      # serviceaccount token or token_file token_file 中应注意不要有换行符 请使用 echo -n token > token_file 来生成此文件
```

Reply:
It's better to show the shell command in a code block.

```shell
echo -n token > token_file
```